migration/v0.40/prepare/20-update-google-dex-config.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+HERE="$(dirname "$(readlink -f "${0}")")"
+ROOT="$(readlink -f "${HERE}/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+tmp_secrets=$(mktemp)
+append_trap "rm ${tmp_secrets}" EXIT
+
+sed "s/^\([ \t]*\)adminEmail: \(.*\)$/\1domainToAdminEmail: \n\1  \'\*\': \2/" <(sops -d "${CK8S_CONFIG_PATH}/secrets.yaml") | sops -e --input-type yaml --config "${CK8S_CONFIG_PATH}/.sops.yaml" /dev/stdin > "${tmp_secrets}"
```

Comment:
Why not output this directly to the secret and skip the temp file?

```suggestion
sed "s/^\([ \t]*\)adminEmail: \(.*\)$/\1domainToAdminEmail: \n\1  \'\*\': \2/" <(sops -d "${CK8S_CONFIG_PATH}/secrets.yaml") | sops -e --input-type yaml --config "${CK8S_CONFIG_PATH}/.sops.yaml" /dev/stdin > "${CK8S_CONFIG_PATH}/secrets.yaml"
```

Reply:
I tried that at first, but it then failed. I think that the piping starts to output things to that secret file before sed is fully done with reading it. Got this error:
```
sops metadata not found                                                                                                                                     
File cannot be completely empty, it must contain at least one document
```